tests/pyccel/test_pyccel.py

```diff
@@ -12,8 +12,8 @@
 #==============================================================================
 
 @pytest.fixture( params=[
-        #pytest.param("fortran", marks = pytest.mark.fortran),
-        #pytest.param("c", marks = pytest.mark.c),
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = pytest.mark.c),
```

Comment:
Just guessing, but if you want to test just the python code on the runner it is better to comment this:
- https://github.com/pyccel/pyccel/blob/master/.github/workflows/Github_pytest.yml#L24-L25
- https://github.com/pyccel/pyccel/blob/master/.github/workflows/Github_pytest.yml#L63-L80
- https://github.com/pyccel/pyccel/blob/master/.github/workflows/Github_pytest.yml#L124-L125